packages/components/src/combobox-control/test/index.tsx

```diff
@@ -18,7 +18,7 @@ import type { ComboboxControlOption, ComboboxControlProps } from '../types';
 const timezones = [
 	{ label: 'Greenwich Mean Time', value: 'GMT' },
 	{ label: 'Universal Coordinated Time', value: 'UTC' },
-	{ label: 'European Central Time', value: 'ECT' },
+	{ label: 'European Central Time', value: 'ETC' },
```

Comment:
This value is an abbreviation for "European Central Time", so no changes should be necessary.

Reply:
@t-hamano This is a halfdone PR.

So do you give a green light?

Reply:
Does this mean this PR is still a work in progress? Could you give us more details?

Reply:
This PR is the result of my default `typos` config + `typos -w` (autofix) + some manual work.
I need your confirmation before working on it for an hour or two.